docker/cluster/master_1/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM redis-py-base:latest
+
+COPY redis.conf /usr/local/etc/redis/redis.conf
+COPY nodes.conf /usr/local/etc/redis/nodes.conf
+
+EXPOSE 7000
+
+RUN chmod 666 /usr/local/etc/redis/nodes.conf
```

Comment:
Why 666? I'd assume 0600, or 0660 at the least.

Reply:
Actually, 006 does the job. I changed it.